test/Pester/PlatyPs.Tests.ps1

```diff
@@ -146,6 +146,63 @@ Describe 'New-MarkdownHelp' {
         }
     }
 
+    Context 'from external script' { 
+        It 'fully qualified path' {
+            $NL = [System.Environment]::NewLine
+            $SeedData = (
+                "<# $NL" +
```

Comment:
I think here-string would be easier here
https://devblogs.microsoft.com/scripting/powertip-use-here-strings-with-powershell/

Reply:
I thought here-strings were frowned upon in code that was already indented since it breaks that indentation? Nonetheless, I can convert if that's your preference.

Reply:
Now converted to here-strings.